src/main/java/at/tugraz/sw20asd/lang/ui/Controller/StudyVocab.java

```diff
@@ -0,0 +1,226 @@
+                public void handle(ActionEvent event) {
+                    OverviewVocabs overview = new OverviewVocabs(vocab);
+                    getScene().setRoot(overview);
+                }
+            });
+
+        }
+
+        private void showAllAnswers(){
+                if(show_all_btn.getText().equals("Show All Answers")){
+                    for(int counter = 0; counter < given_label_list.size(); counter++) {
+                        answer_label_list.get(counter).setVisible(true);
+                        answer_button_list.get(counter).setText("Hide Answer");
+                        show_all_btn.setText("Hide All Answers");
+                    }
+                }
+                else{
+                    for(int counter = 0; counter < given_label_list.size(); counter++) {
+                        answer_label_list.get(counter).setVisible(false);
+                        answer_button_list.get(counter).setText("Show Answer");
+                        show_all_btn.setText("Show All Answers");
+                    }
+                }
+            }
+
+        private void switchLanguage(){
+            show_all_btn.setText("Hide All Answers");
+            showAllAnswers();
+            for(int counter = 0; counter < given_label_list.size(); counter++)
+            {
+                String new_answer = given_label_list.get(counter).getText();
+                String new_given = answer_label_list.get(counter).getText();
+                String new_given_language = study_label.getText();
+                String new_study_language = given_label.getText();
+
+                given_label_list.get(counter).setText(new_given);
+                answer_label_list.get(counter).setText(new_answer);
+                given_label.setText(new_given_language);
+                study_label.setText(new_study_language);
+            }
+        }
+
+        private void populate(){
+            user_info.setVisible(false);
+            show_all_btn.setVisible(true);
+            switch_btn.setVisible(true);
+            given_label.setVisible(true);
+            study_label.setVisible(true);
+            anchor_pane.getChildren().remove(choice_label);
+            anchor_pane.getChildren().remove(language_choice);
+            anchor_pane.getChildren().remove(start_btn);
+
+            int min_height = 30;
+            int margin = 10;
+            for (int counter = 0; counter < words.size(); counter++) {
+
+                String given_str;
+                String answer_str;
+                if(study_language.equals("de")){
+                    given_str = words.get(counter).getTranslation();
+                    answer_str = words.get(counter).getPhrase();
+                }
+                else{
+                    given_str = words.get(counter).getPhrase();
+                    answer_str = words.get(counter).getTranslation();
+                }
+
+                Label given = new Label(given_str);
```

Comment:
Please, consider moving all that styling into the relevant `*.fxml` and just bind models to each of the "study columns".